framework/src/main/java/org/checkerframework/framework/type/DefaultTypeHierarchy.java

```diff
@@ -370,6 +370,11 @@ private boolean isContainedWildcard(
             AnnotatedTypeMirror outsideLowerBound,
             boolean canBeCovariant) {
 
+        if (inside.getKind() == TypeKind.WILDCARD
+                && outside.getKind() == TypeKind.WILDCARD
+                && inside.equals(outside)) {
+            return true;
```

Comment:
Could you add a comment why this is the right check?
Or, as we discussed last week, add a comment to the method or maybe even directly on `AnnotatedWildcardType` that wildcards are used also for captured variables. All logic that uses AWT should be investigated once we have a separate captured type.